clients/adapter/src/main/java/org/eclipse/hono/adapter/client/command/Command.java

```diff
@@ -87,14 +89,20 @@
     /**
      * Gets the device identifier used in the original command. It is extracted from the
      * <em>to</em> property of the command AMQP message.
```

Comment:
Since this is a generic class, IMHO we should remove the reference to the AMQP specific *to* property ...
I would also suggest to make this more explicit:
> Gets the identifier of the device that is supposed to execute the command.